### ISSUE TITLE
rename to `dlwhich`?

### ISSUE THREAD

**Issue by vtjnash** [vtjnash/libwhich#12]
I'm worried the name `libwhich` would mislead folks into thinking this is a library-ized version of `which`, which is not the intended parsing of the name. Perhaps alternate names are `whichlib` or `dlwhich`?

**Comment by staticfloat** [comment]
I like `dlwhich` better, myself. :)

**Comment by jirutka** [comment]
> I'm worried the name libwhich would mislead folks into thinking this is a library-ized version of which, which is not the intended parsing of the name. 

Exactly! `whichlib` (:+1:) or `dlwhich` would be definitely less confusing.

**Comment by oxr463** [comment]
What does the `dl` stand for?

I have no idea if this is possible, but if the binary were to be `which-lib` and then it could be invoked as `which --lib` kind of like a git extension, that would be pretty cool.

**Comment by ararslan** [comment]
> What does the `dl` stand for?

Presumably [dynamic loading](https://en.wikipedia.org/wiki/Dynamic_loading), as in `libdl`.

**Comment by haampie** [comment]
If you ever decide to change the name of the executable, then it would be a good opportunity to default to printing the resolved path only (currently `libwhich -p`) and put the rest under a `-v` flag or so.

`whichlib` is easier to tab complete than `dlwhich` cause I wouldn't remember `dl<tab>` (like libdl) vs `ld<tab>` (like ldd).
